src/server/typesMap.json

```diff
@@ -22,6 +22,10 @@
         "References": {
             "match": "^(.*\\/_references\\.js)$",
             "exclude": [["^", 1, "$"]]
+        },
+        "Datatables.net": {
+            "match": "(jquery\\.)?dataTables(\\.all)?(\\.min)?\\.js$",

```

Comment:
This RegExp would also match `mydataTables.js`. There's a reason all the others uses a `^.*\/`-like pattern at the start, please do similar.

Also note all the other patterns are lower-cased. I believe the path is normalized to lowercase before the regexp is applied. Please follow this pattern also, else it may not match due to casing.

Reply:
Fixed. Notice that the jquery library is also missing that same pattern. Let me know if that is expected or should be fixed.